source/options.html

```diff
@@ -45,6 +45,16 @@ <h2 class="only-firefox">Options</h2>
 
 	<hr>
 
+	<p>
+			<label>
+				<strong>Muted users</strong>
```

Comment:
The feature that does this needs to be clear; perhaps:

> Users muted by `muted-users` feature 

Ideally the field would appear in the feature list, but I think that’s a little more complicated. Perhaps the field can be moved from here to the list via JS, right after the list is generated. 

The label should then just be “Users”

Reply:
![image](https://user-images.githubusercontent.com/37769974/59781606-c3ad8c00-92d9-11e9-95eb-4a68903f05d0.png)

???

Reply:
Seems redundant. Either Users or “User list” I think would suffice

Reply:
![image](https://user-images.githubusercontent.com/37769974/59837486-f5bafe80-936a-11e9-939f-3e04d416ff45.png)